Twig/Extension/TextFormatterExtension.php

```diff
@@ -50,7 +52,7 @@ public function getTokenParsers()
     public function getFilters()
     {
         return array(
-            'format_text' => new \Twig_Filter_Method($this, 'transform'),
+            'format_text' => new \Twig_SimpleFilter($this, 'transform'),
```

Comment:
So let's take profit to improve the requirement on composer: https://github.com/sonata-project/SonataFormatterBundle/blob/ae7b4084a14f24348537bd58a62e0a06d0d2efb2/composer.json#L23

;-)